.dockerignore

```diff
@@ -0,0 +1,27 @@
+.git
+.idea
+.idea/
+__pycache__/
+.gitignore
+LICENSE
+VERSION
+README.md
+Changelog.md
+Makefile
+docker-compose.yml
+docs
+.ipynb_checkpoints/*
+/notebooks/*
+/unused/*
+Dockerfile
+.DS_Store
```

Comment:
do we need all this?

Reply:
I will remove unnecessary files. You are right